frontend/javascripts/dashboard/dataset/dataset_settings_data_tab.tsx

```diff
@@ -138,37 +167,70 @@ function SimpleDatasetForm({
         }
       >
         <List.Item>
-          <FormItemWithInfo
-            name={["dataSource", "scale"]}
-            label="Voxel Size"
-            info="The voxel size defines the extent (for x, y, z) of one voxel in nanometer."
-            initialValue={dataSource != null ? dataSource.scale : [0, 0, 0]}
-            rules={[
-              {
-                required: true,
-                message: "Please provide a scale for the dataset.",
-              },
-              {
-                validator: syncValidator(
-                  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'el' implicitly has an 'any' type.
-                  (value) => value && value.every((el) => el > 0),
-                  "Each component of the scale must be larger than 0",
-                ),
-              },
-            ]}
+          <Row
+            gutter={48}
+            style={{
+              width: "100%",
+            }}
           >
-            <Vector3Input
-              disabled={isReadOnlyDataset}
-              style={{
-                width: 400,
-              }}
-              allowDecimals
-            />
-          </FormItemWithInfo>
+            <Col span={10}>
+              <FormItemWithInfo
+                name={["dataSource", "id", "name"]}
+                label="Name"
+                info="The name of the dataset"
+                rules={[
+                  {
+                    required: true,
+                    message: "Please provide a name for the dataset.",
+                  },
+                  {
+                    validator: syncValidator((value) => value.length > 3, "..."),
```

Comment:
@fm3 Do you know what the constraints are for the name of a dataset?

Reply:
same as for dataset upload https://github.com/scalableminds/webknossos/blob/master/app/models/binary/DataSetService.scala#L53
```
      _ <- bool2Fox(name.matches("[A-Za-z0-9_\\-\\.]*")) ?~> "dataSet.name.invalid.characters"
      _ <- bool2Fox(!name.startsWith(".")) ?~> "dataSet.name.invalid.startsWithDot"
      _ <- bool2Fox(!name.endsWith(".")) ?~> "dataSet.name.invalid.endsWithDot"
      _ <- bool2Fox(name.length >= 3) ?~> "dataSet.name.invalid.lessThanThreeCharacters"
```

Reply:
Great, thank you. I found the frontend code which already validated this for the dataset name in the other import view. However, it didn't include the `.` rules. I assume that this is a bit newer and the frontend and backend diverged on this? Maybe you can add a comment to the back-end that this is also checked by front-end (it's a bit redundant, but I find it good for better readable messages).

Reply:
Yes, the dot rules are newer. I’m not sure anymore if the endsWith dot should be removed again. What do you think? The startsWith dot is to avoid the dataset becoming a hidden directory

Reply:
> I’m not sure anymore if the endsWith dot should be removed again. What do you think?

I don't know whether there could be problems with zarr routes because the `.` could be a delimiter there? However, a dot in the middle of the name also seems to be supported at the moment, so a trailing `.` might also be ok. Other than the zarr topic, I don't see a reason to forbid trailing dots.

Reply:
Then let’s just undo that (do you want to do this here and remove the line in the backend?)

Reply:
done in backend and frontend :)